SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java

```diff
@@ -110,4 +113,12 @@
     Context getContext();
 
     C getConfig();
+
+    List<SwrveBaseCampaign> getCampaigns();
+
+    List<SwrveBaseCampaign> getCampaigns(SwrveOrientation orientation);
```

Comment:
I was instructed to not use the word Inbox as this feature is not really inbox, no UI yet :S